File: android/src/main/java/com/pinmi/react/printer/adapter/abstracts/PrinterDevice.java
package com.pinmi.react.printer.adapter.abstracts;

import com.facebook.react.bridge.WritableMap;

/**
 * Created by xiesubin on 2017/9/21.
 */

public interface PrinterDevice {

    public PrinterDeviceId getPrinterDeviceId();
    public WritableMap toRNWritableMap();

}

File: android/src/main/java/com/pinmi/react/printer/adapter/abstracts/PrinterDevice.kt
package com.pinmi.react.printer.adapter.abstracts

import com.facebook.react.bridge.WritableMap

/**
 * Created by misaeldossantos on 2023/02/25.
 */
interface PrinterDevice {
    val printerDeviceId: PrinterDeviceId
    fun toRNWritableMap(): WritableMap
}
